### PR TITLE
refactor: move config to configs service

### PR DESCRIPTION
**Pull request by austin-king**
## High Level Overview of Change

Just moving config values into a config service to manage our state in a centralized location. @hbergren I copied over some of the linting config (in the initial commit, not this PR). Would prob be worth glancing over to see if there are any issues with it.

- [x] Refactor (non-breaking change that only restructures code)

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -1,17 +1,17 @@
+import Config from './services/config'
+
 import Express = require('express')
+import Reduct = require('reduct')
```
Comment (hbergren) [inline src/app.ts]: What is `Reduct`?

File: src/services/config.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export default class Config {
+  public privateApiPort: number
+
+  public publicApiPort: number
+
+  constructor() {
+    this.publicApiPort = 8080
+    this.privateApiPort = 8081
+  }
```
Comment (hbergren) [inline src/services/config.ts]: You might be able to use class fields to define the numbers right above, like:

```js
class Config {
 publicApiPort = 8080
 privateApiPort = 8081
}
```

If not, could you switch the order of the class members to match the order in the constructor?

File: src/app.ts (hunk 1 above)
Comment (austin-king) [reply src/app.ts]: It's a functional dependency manager, I basically use it to force myself to keep my services stateless and manage all that in a config file. It's just a tool I've gotten used to using, happy to remove if you think would be confusing to other maintainers. Just lmk.

File: src/services/config.ts (hunk 2 above)
Comment (austin-king) [reply src/services/config.ts]: Good point, pushed fix.

**Pull request merged**